java/system/ksp-allow-springboot-conf-file.yaml

```diff
@@ -0,0 +1,56 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-allow-springboot-conf-file
+  namespace: default
+spec:
+  tags: ["springboot", "framework", "configuration","tomcat"]
+  message: "Alert! The configuration files are permited"
```

Comment:
Can you change message to something more related to the policy? like allow access to the files from java binary cc: @raviknox